test/checks/visibility/hidden-content.js

```diff
@@ -1,46 +1,81 @@
+/* global xit */
 describe('hidden content', function () {
 	'use strict';
 
-  var fixture = document.getElementById('fixture');
-
+	var fixture = document.getElementById('fixture');
+	var shadowSupport = document.body && typeof document.body.attachShadow === 'function';
 	var checkContext = {
 		_data: null,
 		data: function (d) {
 			this._data = d;
 		}
 	};
 
+	function checkSetup (html, options, target) {
```

Comment:
Should we abstract this into a reusable utility so it doesn't have to get repeated in every test file needing Shadow DOM support?